ShopApp.xcodeproj/project.pbxproj

```diff
@@ -3171,10 +3298,13 @@
 				0B5768DA204D3B040001738D /* SignUpViewControllerSpec.swift in Sources */,
 				0B83E00A20504AA10037035D /* GridCollectionViewControllerSpec.swift in Sources */,
 				0BE965B92045B10400308CA9 /* GridCollectionProviderDelegateMock.swift in Sources */,
+				0B83E02B2056A4470037035D /* ArticleDetailsViewModelSpec.swift in Sources */,
 				0B5768D4204D3B040001738D /* OrderDetailsViewControllerSpec.swift in Sources */,
 				0B43717A204D783F00C086CF /* OrderFooterDelegateMock.swift in Sources */,
 				0BE96570203FF28A00308CA9 /* SignInViewModelMock.swift in Sources */,
 				614C80D220470381006A99E0 /* AccountAddressFormViewModelSpec.swift in Sources */,
+				0B83E01920566C1B0037035D /* ArticleRepositoryMock.swift in Sources */,
+				0B83E020205673320037035D /* ArticlesListTableProviderSpec.swift in Sources */,
```

Comment:
Files with 'list' word should named with single plural form (like ArticleListTableProviderSpec)